starter/src/main/java/de/fraunhofer/iosb/ilt/faaast/service/starter/App.java

```diff
@@ -387,12 +387,15 @@ private void runService(ServiceConfig config) {
     private Optional<File> findDefaultModel() {
         try {
             List<File> modelFiles;
+            List<String> fileExtensions = new ArrayList<>(DataFormat.AASX.getFileExtensions());
+            fileExtensions.addAll(DataFormat.JSON.getFileExtensions());
             try (Stream<File> stream = Files.find(Paths.get(""), 1,
                     (file, attributes) -> file.toFile()
                             .getName()
                             .matches(MODEL_FILENAME_PATTERN))
                     .map(Path::toFile)) {
-                modelFiles = stream.collect(Collectors.toList());
+                modelFiles = stream.filter(f -> fileExtensions.stream().anyMatch(f.getName()::endsWith))
```

Comment:
Wouldn't `.endswith` also allow incorrect file extensions like e.g. `.fooaasx`?
We should do an exact match on the file extension, maybe this utility method might help here https://github.com/FraunhoferIOSB/FAAAST-Service/blob/main/core/src/main/java/de/fraunhofer/iosb/ilt/faaast/service/util/FileHelper.java#L115

Reply:
True, I changed it to a comparison with the FileHelper